README.md

```diff
@@ -1,141 +1,146 @@
 
-When a new court is initialized, there will be a small amount of reputation relative to collateral, but overtime as the court adjudicates disputes the proportion of reputation relative to collateral will increase. As the court matures, this transition makes it less and less cost effective to influence the rulings of a court by purchasing and staking collateral.
+Jurors are given a certain number of terms, called the *commitment period*, to submit a hash of their ruling. After the commit period ends, jurors are given a certain number of terms to reveal their ruling, this period is called the *reveal period*. If a juror reveals their vote prior to the reveal period, anyone can use this information to penalize the juror in exchange for a reward. If a juror fails to commit and reveal a vote by the end of the *reveal period* they are penalized.
 
-Since reputation is court specific and non-transferrable it is more difficult to sell, however, even if reputation within a court is sold, the impact of an individual court becoming compromised is minimal. Honest jurors within the court can organize to fork the court keeping their reputation balances, and parties of agreements enforced by the court can mutually agree to use a different court.
-## 3.2 Adjudication Process
-We assume that a client side application will be used by all participants to monitor on-chain changes and alert them when actions are required. This includes being selected for a jury, dispute initialization, appeals, and ultimately resolution.
+After the conclusion of *reveal period*, a majority of support among drafted jurors is used as a *preliminary ruling*. The preliminary ruling is considered final if there is no appeal made during the *appeal period*.  
 
-Jurors who are willing to actively participate in arbitration within a court activate their reputation in order to be eligible for selection on a jury. When a user triggers a dispute they must pay an arbitration fee proportional to the amount of reputation that will be included on the jury. A jury is formed by randomly selecting from all activated reputation within the court at the time of the dispute.
+### Appeals
 
-Once the dispute is initialized and the jury formed, the adjudication process is broken up into four time windows, the duration of each can be different depending on the specific court being used:
+Each dispute is subject to a maximum number of appeal rounds. Since each dispute and appeal round has a fixed duration, the max appeal rounds also determines the maximum amount of time before a final decision is reached.
```

Comment:
```suggestion
Each dispute is subject to a maximum number of appeal rounds. Since each dispute and appeal round has a fixed duration, the maximum number of appeal rounds also determines the maximum amount of time before a final decision is reached.
```